SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -127,6 +133,11 @@ public SingularityExecutorConfiguration(
     this.logrotateCount = logrotateCount;
     this.logrotateMaxageDays = logrotateMaxageDays;
     this.logrotateDateformat = logrotateDateformat;
+    if ((s3FilesToBackup != null) && (s3FilesToBackup.trim().length() > 0)) {
+      this.s3FilesToBackup = s3FilesToBackup.split(",");
+    } else {
+      this.s3FilesToBackup = new String[0];
+    }
```

Comment:
using a `Splitter` would probably be cleaner here (logrotateExtrasFiles is using this same pattern, maybe we should change that too)